### PR TITLE
returning classifications from post and delete

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

File: src/services/employees.js [hunk 1]
```diff
@@ -498,15 +498,13 @@ const get_classifications = async query => {
...
   const perdet_seq_num = query['perdet_seq_num']
-  
   try {
     if (Array.isArray(tracking_events)) {
       const proms = tracking_events.map(async (tracking_event) => {
         await EmployeesClassifications.forge({
           te_id: tracking_event,
           perdet_seq_num: perdet_seq_num,
         }).save()
-        return
```
Comment (elizabeth-jimenez) [inline src/services/employees.js:509]: ☠️

File: src/services/employees.js [hunk 2]
```diff
@@ -532,7 +530,6 @@ const remove_classification = async query => {
           te_id: tracking_event,
           perdet_seq_num: perdet_seq_num,
         }).destroy()
-        return
```
Comment (elizabeth-jimenez) [inline src/services/employees.js:535]: 💀

File: src/services/employees.js (hunk 1 above)
Comment (scott062) [reply src/services/employees.js:509]: Is this something we were messing with b/c of the async stuff - I can't remember now?

Comment (elizabeth-jimenez) [reply src/services/employees.js:509]: i feel like it was, yea

**Review by scott062 — APPROVED** [review]
Was able to add single/multiple 👍  also was able to see the current classifications on post/delete in res

**Pull request merged**